lib/sheriff/plug/enforce_policy.ex

```diff
@@ -25,7 +25,15 @@ defmodule Sheriff.Plug.EnforcePolicy do
   end
 
   defp fetch_actor(conn) do
-    {conn.private[:current_user], conn}
+    case System.get_env("SHERIFF_RESOURCE_KEY") do
```

Comment:
The library should rely on `Application.get_env` and not `System.get_env` which doesn't play nicely with releases.  If a user wants to rely on `System.get_env` they can set it up in their config.